src/main/java/com/github/technus/tectech/rendering/EOH/EOH_RenderingUtils.java

```diff
@@ -0,0 +1,265 @@
+package com.github.technus.tectech.rendering.EOH;
+
+import static com.github.technus.tectech.Reference.MODID;
+import static com.github.technus.tectech.rendering.EOH.EOH_TESR.STAR_LAYER_0;
+import static com.github.technus.tectech.rendering.EOH.EOH_TESR.STAR_LAYER_1;
+import static com.github.technus.tectech.rendering.EOH.EOH_TESR.STAR_LAYER_2;
+import static com.github.technus.tectech.rendering.EOH.EOH_TESR.spaceModel;
+import static com.github.technus.tectech.rendering.EOH.EOH_TESR.starModel;
+import static java.lang.Math.pow;
+
+import java.awt.Color;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.util.IIcon;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.IItemRenderer;
+
+import org.lwjgl.opengl.GL11;
+
+import cpw.mods.fml.client.FMLClientHandler;
+
+public abstract class EOH_RenderingUtils {
+
+    public static void renderStar(IItemRenderer.ItemRenderType type, Color color) {
+        GL11.glPushMatrix();
+
+        if (type == IItemRenderer.ItemRenderType.INVENTORY) GL11.glRotated(180, 0, 1, 0);
+        else if (type == IItemRenderer.ItemRenderType.EQUIPPED
+                || type == IItemRenderer.ItemRenderType.EQUIPPED_FIRST_PERSON) {
+                    GL11.glTranslated(0.5, 0.5, 0.5);
+                    if (type == IItemRenderer.ItemRenderType.EQUIPPED) GL11.glRotated(90, 0, 1, 0);
+                }
+
+        // Render star stuff.
+        renderStarLayer(0, STAR_LAYER_0, color, 1.0f);
+        renderStarLayer(1, STAR_LAYER_1, color, 0.4f);
+        renderStarLayer(2, STAR_LAYER_2, color, 0.2f);
+
+        GL11.glPopMatrix();
+    }
+
+    public static void renderStar(IItemRenderer.ItemRenderType type) {
+        renderStar(type, new Color(1.0f, 0.4f, 0.05f, 1.0f));
+    }
+
+    private static void renderStarLayer(int layer, ResourceLocation texture, Color color, float alpha) {
+
+        // Begin animation.
+        GL11.glPushMatrix();
+
+        // OpenGL settings, not sure exactly what these do.
+
+        // Disables lighting, so star is always lit (I think).
+        GL11.glDisable(GL11.GL_LIGHTING);
+        // Culls things out of line of sight?
+        GL11.glEnable(GL11.GL_CULL_FACE);
+        // Merges colours of the various layers of the star?
+        GL11.glEnable(GL11.GL_BLEND);
+        // ???
```

Comment:
this is a bit weird, idk why it's conditional on alpha, but if it achieves the desired result that's fine